lib/core/utils/parse-crossorigin-stylesheets.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Parse cross-origin stylesheets
+ *
+ * @param {String} url url from which to fetch stylesheet
+ * @param {Object} options options object from `axe.utils.parseStylesheets`
+ * @param {Array<Number>} priority sheet priority
+ * @param {Array<String>} importedUrls urls of already imported stylesheets
+ * @param {Boolean} isCrossOrigin boolean denoting if a stylesheet is `cross-origin`
+ * @returns {Promise}
+ */
+axe.utils.parseCrossOriginStylesheets = function parseCrossOriginStylesheets(
+	url,
+	options,
+	priority,
+	importedUrls,
+	isCrossOrigin
```

Comment:
Is this param needed? I would assume it would always be `true` since the function is specifically used to load and parse cross origin stylesheets.

Reply:
This is why we need it - https://github.com/dequelabs/axe-core/pull/1339#discussion_r278497604